python/whylogs/core/validators/condition_validator.py

```diff
@@ -0,0 +1,52 @@
+from dataclasses import dataclass
+from itertools import chain
+from typing import Any, Callable, Dict, List
+
+from whylogs.core.metrics.condition_count_metric import Condition
+from whylogs.core.metrics.metric_components import IntegralComponent
+from whylogs.core.preprocessing import PreprocessedColumn
+from whylogs.core.validators.validator import Validator
+
+
+@dataclass
+class ConditionValidator(Validator):
+    def __init__(self, name: str, conditions: Dict[str, Condition], actions=List[Callable]) -> None:
+        self.conditions = conditions
+        self.name = name
+        self.actions = actions
+        self.total: IntegralComponent = IntegralComponent(0)
```

Comment:
Why not plain integers?

Reply:
Actually, I might make `total_evaluations` and `total_failures` dictionaries keyed by the condition name.

Reply:
Yes, you're right, I carried `IntegralComponent` from the Condition Count implementation, but it's not needed here. I'll remove it.

I agree that maybe we don't need `total_failures`. If only `total_evaluations` is left, do we still need to separate it into another dictionary?

My idea is to make it like:
```
{
"total_evaluations": 75,
"condition#1": 2,
"condition#2": 1,
.
.
.
}
```

Reply:
I changed it as per my last comment. resolving this comment, if needed we can reopen it.